### ISSUE TITLE
License checklist

### ISSUE THREAD

**Issue by jianructose** [UBC-MDS/rgtracker#27]
Examine the license for your project and consider whether this is the choice you want to make, or whether you want to change the license. Discuss and reason the license choice by opening issues in both Python and R repositories. As it is likely to be a very similar discussion for both projects, one of these issues can just link to the other issue where it is thoroughly dicussed.

**Comment by tdkhanhvu** [comment]
I think we will ignore the below:
- Mozilla Public License 2.0
- Apache License 2.0

As we do not put an emphasis on limiting the **Trademark use**

**Comment by jraza19** [comment]
In my opinion, I think the MIT License works best. For future use, since our functions are simple and could be modified for other applications, one of us (along with other people) may come back to this code to use for later and so keeping it simple and being open and permissive such as MIT License sounds good to me!

**Comment by yhchen20** [comment]
@jraza19 I agree. I would like to have such a permissive license, too. However,  I think all the changes in the materials should be documented, so I prefer to use GNU GPLv3 License

**Comment by jianructose** [comment]
> I think we will ignore the below:
> 
> * Mozilla Public License 2.0
> * Apache License 2.0
> 
> As we do not put an emphasis on limiting the **Trademark use**

Yes. Agree. I would vote for GNU GPLv3 or GNU LGPLv3, since they condition on changes to any licensed material must be documented. This had aligned with our code of conduct as well.

**Comment by jraza19** [comment]
@jianructose I think that GNU LGPLv3 version seems more lenient from the other GPU licenses versions! Perhaps we should go with that one!

**Comment by tdkhanhvu** [comment]
> @jianructose I think that GNU LGPLv3 version seems more lenient from the other GPU licenses versions! Perhaps we should go with that one!

Yes, I agree with this

**Comment by yhchen20** [comment]
> @jianructose I think that GNU LGPLv3 version seems more lenient from the other GPU licenses versions! Perhaps we should go with that one!

Agree
